src/styles/abstracts/mixin.tsx

```diff
@@ -0,0 +1,45 @@
+export const fontMontserratRegular = () => {
+  return `
+    font-family: "Montserrat", Helvetica, Arial, sans-serif;
+    font-weight: 400;
+  `;
+};
+
+export const fontMontserratMedium = () => {
+  return `
+    font-family: "Montserrat", Helvetica, Arial, sans-serif;
+    font-weight: 500;
+  `;
+};
+
+export const fontMontserratSemibold = () => {
+  return `
+    font-family: "Montserrat", Helvetica, Arial, sans-serif;
+    font-weight: 600;
+  `;
+};
+
+export const fontSourcesansproRegular = () => {
+  return `
+    font-family: "Source Sans Pro", Helvetica, Arial, sans-serif;
+    font-weight: 400;
+  `;
+};
+
+export const fontSourcesansproBold = () => {
+  return `
+    font-family: "Source Sans Pro", Helvetica, Arial, sans-serif;
+    font-weight: 700;
+  `;
+};
+
+const pxRem = (size: number, base = 16) => {
```

Comment:
pxToRem ?

Reply:
helps to convert px value to correct rem values. eg:

10px = 0.625rem
16px = 1rem

Reply:
yes usually those functions are name aToB